drafts/25-114.txt

```diff
@@ -239,8 +239,6 @@ FPP will not expand in fixed-form
 
 FPP will not expand tokens in either fixed- or free-form:
     - In character constants
```

Comment:
The standard uses the phase "character literal constant" in 7.4.4.3, and I think we should echo that here.

Also do we need to also explicitly mention character string (ie quoted) edit descriptors?

IIUC those are not technically character literal constants (see 13.9), but we also want to prohibit expansion there (partially because the preprocessor lacks context information to discern whether a quoted character string token appears in an edit descriptor).

```suggestion
    - In character literal constants
    - In character string edit descriptors (quoted character strings) 
```

Reply:
I agree with Dan.

Reply:
I agree with Steve and Dan. Update forthcoming.